src/main/java/org/jenkinsci/plugins/webhookstep/RegisterWebhookExecution.java

```diff
@@ -6,9 +6,15 @@
 public class RegisterWebhookExecution extends AbstractSynchronousStepExecution<WebhookToken> {
 
     private static final long serialVersionUID = -6718328636399912927L;
+    private final String authToken;
 
-    public RegisterWebhookExecution(StepContext context) {
+    public RegisterWebhookExecution(StepContext context, String authToken) {
```

Comment:
When adding new parameters, it's best to use an @DataBoundSetter instead of adding a parameter to the constructor, this way binary compatibility is not broken.